java/src/main/java/org/rocksdb/MemoryUsageType.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+/**
+ * MemoryUsageType
+ *
+ * <p>The value will be used as a key to indicate the type of memory usage
+ * described</p>
+ */
+public enum MemoryUsageType {
```

Comment:
Done and switched to using toJava for the enum -> jbyte conversion.

Using bytes here since I don't see us going > 256 entries.

If you disagree with where I put it in portal.h let me know, I didn't see a strong sort order there.